Lab 2/token.in

```diff
@@ -0,0 +1,29 @@
++
+-
+*
+/
+%
+<
+<=
+=
+>=
+==
```

Comment:
"<>" appears in _Syntax.in_ but not in _token.in_

Reply:
Quite the oversight on my part, thank you!